### ISSUE TITLE
Missing Link

### ISSUE THREAD

**Issue by pwilliv** [MicrosoftDocs/windows-itpro-docs#10657]
The mentioned Table "Tabelle 13. Dateizugriffscodes." of Accesscodes is not linked.
This would be helpful.

(admin edit: switching to the en-us source file for preferred tracking)

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 7cb8130d-57c1-beb4-8ae7-cccfda489627
* Version Independent ID: 5a9afa7b-2745-9b86-8a4e-03bf2294053c
* Content: [5140(S, F) A network share object was accessed. (Windows 10) - Windows security](https://docs.microsoft.com/en-us/windows/security/threat-protection/auditing/event-5140)
* Content Source: [windows/security/threat-protection/auditing/event-5140.md](https://github.com/MicrosoftDocs/windows-itpro-docs/blob/public/windows/security/threat-protection/auditing/event-5140.md)
* Product: **m365-security**
* Technology: **windows-sec**
* GitHub Login: @Dansimp
* Microsoft Alias: **dansimp**

**Comment by aczechowski** [comment]
For reference:

> **Access Request Information:**
>
> - **Access Mask** [Type = HexInt32]: the sum of hexadecimal values of requested access rights. See “Table 13. File access codes.” for different hexadecimal values for access rights. Has always “0x1” value for this event.

**Comment by yogkumgit** [comment]
@pwilliv Hope this comment is helpful for you. If you see a documentation update is required, please feel free to open an issue for the same. We proceed here to close it. Thanks for taking out some time to open the issue. Appreciate and encourage you to do the same in future also.

**Comment by aczechowski** [comment]
@yogkumgit there is a change to be made here (and actually in a couple of articles). There is a reference to "see table 13" that's not linked, and there's no "table 13" in this article. I searched and found it in another article: https://learn.microsoft.com/en-us/windows/security/threat-protection/auditing/event-5145#table-of-file-access-codes

As I hinted, there are multiple such references, so it would be great to get them all fixed:

- event-4661.md
  - line 166
  - line 168
- event-4691.md
  - line 128
  - line 130
- event-5140.md (the article flagged in this issue), line 135
- event-5145.md, line 137

At the same time, would be great to fix the style of the cross-reference. My suggestion is to replace the existing sentences to "see 'Table 13. File access codes" to use our standard linking format, for example:

```markdown
For more information, see [Table of file access codes](/windows/security/threat-protection/auditing/event-5145#table-of-file-access-codes).
```

(Some specific instances may require slightly different wording, but can use that same general style.)
Let me know if any questions! Thank you :)
